src/main/java/com/bbd/modisa/client/BankApp.java

```diff
@@ -15,9 +12,9 @@
     private static Scanner scanner = new Scanner(System.in);
     private static Double amount;
     private static int accountNo = 0;
+    private static AccountServiceFactory accountServiceFactory = new AccountServiceFactory();
```

Comment:
We should be using this only in one place i.e where we are going to create an accountService.So it should not be a member variable.